lib/collection/src/operations/point_ops.rs

```diff
@@ -23,6 +23,17 @@ pub struct PointStruct {
     pub payload: Option<Payload>,
 }
 
+/// Warn: panics if the vector is empty
+impl From<Record> for PointStruct {
+    fn from(record: Record) -> Self {
+        PointStruct {
+            id: record.id,
+            vector: record.vector.unwrap(),
+            payload: record.payload,
+        }
+    }
+}
```

Comment:
Why not `TryFrom` then?

Reply:
I tried `TryFrom` but it was too complicated to propagate the error properly. It was used inside the `.map`, so I should have use some strange constructions

Reply:
For iterating over errors we can do something like this - https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=53573a8d95b0813698dde7486021e00b

But I am not against leaving it like it is